include/MySugar/MySugar.php

```diff
@@ -80,8 +80,9 @@ public function checkDashletDisplay()
             $bean = SugarModule::get($this->type)->loadBean();
             if (!ACLController::checkAccess($this->type, 'list', true, $bean->acltype)) {
                 $displayDashlet = false;
+            }else{
+                $displayDashlet = true;
```

Comment:
This is just a suggestion, but I would simplify the function by moving the `$displayDashlet = true;` to the top of the function, at the every beginning, and then remove the two elses.